documentation/build-a-marketplace/README.md

```diff
@@ -19,9 +19,9 @@ Using Ocean Market is already a big improvement on the alternatives that are out
 
 The tutorial covers:
 
-* [Forking and running Ocean Market locally](forking-ocean-market.md)
-* [Customizing your fork of Ocean market](customising-your-market.md)
-* [Quick deployment of Ocean Market](deploying-market.md)
```

Comment:
These links to the other files are useful, I would suggest keeping them. Just double check they work